### PR TITLE
BOCM-357 seed for incomplete & overdue appeals added

### PR DESCRIPTION
**Pull request by MariaValeroGonzalezKC**
## Type of change 🧩

- [√ ] Bug fix (non-breaking change which fixes an issue)
- [ √] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [√ ] Documentation update
- [ ] Other (please explain in the description section above)

## Checklist before requesting a review

- [√ ] I have performed a self-review of my own code
- [√ ] I have double checked this work does not include any hardcoded secrets or passwords
- [√ ] I have made corresponding changes to the documentation
- [√ ] I have provided details on how I have tested my code
- [√ ] I have referenced the ticket number above
- [√ ] I have provided a description of the ticket
- [√ ] I have included unit tests to cover any testable code changes

### CODE REVIEW

**Review by mayya-bond-kc — APPROVED** [review]

File: apps/api/src/server/app/case-officer/case-officer.controller.js [hunk 1]
```diff
@@ -18,6 +19,7 @@ const caseOfficerStatusesOnceQuestionnaireReceived = new Set([
 
 const getAppeals = async function (_request, response) {
 	const appeals = await appealRepository.getByStatusesWithAddresses(caseOfficerStatuses);
+	console.log('HERE ', appeals);
```
[the remark this answers is not in the document]
Comment (MariaValeroGonzalezKC) [reply apps/api/src/server/app/case-officer/case-officer.controller.js]: Done!

**Pull request merged**